module/apmgormv2/apmgorm_test.go

```diff
@@ -121,12 +121,10 @@ func testWithContext(t *testing.T, dsnInfo apmsql.DSNInfo, dialect gorm.Dialecto
 		assert.NotEmpty(t, span.Context.Database.Statement)
 		assert.Equal(t, "sql", span.Context.Database.Type)
 		assert.Equal(t, dsnInfo.User, span.Context.Database.User)
-		if dsnInfo.Address == "" {
-			assert.Nil(t, span.Context.Destination)
-		} else {
-			assert.Equal(t, dsnInfo.Address, span.Context.Destination.Address)
-			assert.Equal(t, dsnInfo.Port, span.Context.Destination.Port)
-		}
+		assert.Equal(t, dsnInfo.Address, span.Context.Destination.Address)
+		assert.Equal(t, dsnInfo.Port, span.Context.Destination.Port)
+		assert.Equal(t, dsnInfo.User, span.Context.Destination.Service.Name)
```

Comment:
```suggestion
		assert.Equal(t, dsnInfo.Database, span.Context.Destination.Service.Name)
```

Right?

Reply:
This is actually destination service name, the old deprecated field. It should use the driver name.